cookbook/showing-authentication-view/README.md

```diff
@@ -0,0 +1,71 @@
+# Showing Authentication View
+
+When your App requires authentication, you should invite the user to login when he tries to access your app features.
+The `SignIn` App provides a signin form ready to be invoked from other `Starcounter` apps thanks to the `Blending` approche.
```

Comment:
Btw, look how nice Grammarly reports the typos:

<img width="708" alt="screen shot 2017-07-28 at 13 49 41" src="https://user-images.githubusercontent.com/566463/28716117-aa8b5b12-739b-11e7-93a8-b3e62e452f5d.png">